CMakeLists.txt

```diff
@@ -132,6 +132,15 @@ include(SetupSphinx)
 include(CodeCoverageDetection)
 include(SpectreAddLibraries)
 
+option(RUN_TESTS_WITH_CHARMRUN
```

Comment:
perhaps make this option `SPECTRE_TEST_RUNNER`, which people can set to things like `charmrun` or `srun` or `mpiexec`? Either way, please add this option to the docs.